Scripts/Button.gd

```diff
@@ -0,0 +1,37 @@
+extends Button
+
+enum eButton { ONHOVER, FINISHED}
```

Comment:
There are 4 states a button can be in: idle, clicked, hovered, and finished right?